pkgs/misc/vim-plugins/update.py

```diff
@@ -491,9 +499,7 @@ def main() -> None:
     prefetch_with_cache = functools.partial(prefetch, cache=cache)
 
     try:
-        # synchronous variant for debugging
-        # results = list(map(prefetch_with_cache, plugin_names))
-        pool = Pool(processes=30)
+        pool = Pool(processes=args.proc)
         results = pool.map(prefetch_with_cache, plugin_names)
```

Comment:
Please add an conditional to not use the pool if `processes == 1`. The pool creates background threads which are harder to debug.

Reply:
```
if processes == 1:
   results = list(map(prefetch_with_cache, plugin_names))
else:
   pool = Pool(processes=args.proc)
   results = pool.map(prefetch_with_cache, plugin_names)
```

Reply:
> The pool creates background threads which are harder to debug.

How is a single pooled process harder to debug?

Reply:
It creates a thread. The other one is just a function call.

Reply:
But how is a single thread harder to debug? `pdb` still works...